features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/mysql.sql

```diff
@@ -79,11 +81,11 @@ CREATE INDEX IDX_AT_CK_AU ON IDN_OAUTH2_ACCESS_TOKEN(CONSUMER_KEY_ID, AUTHZ_USER
 
 CREATE INDEX IDX_TC ON IDN_OAUTH2_ACCESS_TOKEN(TIME_CREATED);
 
-CREATE INDEX IDX_AT ON IDN_OAUTH2_ACCESS_TOKEN(ACCESS_TOKEN);
```

Comment:
Why are we removing existing indexes ?

Reply:
With the introduction of hash column, the search operations will be done using the hash value.
Therefore removed the old  index